docs/apache-airflow-providers-amazon/auth-manager/manage/index.rst

```diff
@@ -18,3 +18,205 @@
+
+.. note::
+  If you manage users in AWS Managed Microsoft AD, make sure that the IAM Identity Center console is using the AWS
+  Region where your AWS Managed Microsoft AD directory is located before taking the next step.
+
+2. Choose **Applications**.
+3. Choose the **Customer managed** tab.
+4. In the list of applications, choose the application name ``Airflow``.
+5. On the application details page, in the **Assigned users and groups** section, choose **Assign users and groups**.
+6. In the **Assign users or groups** page, select the different users and groups you want to assign to Airflow.
+   You can also search users and groups.
+   You can specify multiple users or groups by selecting the applicable accounts as they appear in search results.
+7. Choose **Assign users**.
+
+Manage user permissions through Amazon Verified Permissions
+===========================================================
+
+AWS auth manager uses Amazon Verified Permissions to define and verify user permissions.
+It uses `Cedar language <https://www.cedarpolicy.com/>`__ to define fine-grained permissions for users.
+AWS auth manager uses one policy store to store all policies related to the Airflow environment.
+To manage these policies, please follow the steps below.
+
+1. Open the `Amazon Verified Permissions console <https://console.aws.amazon.com/verifiedpermissions>`_.
+2. Choose the policy store used by Airflow (by default its description is ``Airflow``).
+3. In the navigation pane on the left, choose **Policies**.
+4. On the **Policies** page, you can see the list of policies. To create a new policy, please follow steps below.
+
+  a. Choose **Create policy**.
+  b. Under **Create policy**, choose **Create static policy**.
+
+Policy format
+-------------
+
+Policies are defined using Cedar language in Amazon Verified Permissions.
+For a complete documentation, `see cedar language website <https://www.cedarpolicy.com/>`__.
+In cedar language, a policy is composed of three elements:
+
+* **Principal**. Who is making the request?
+* **Action**. What operation does the principal want to perform?
+* **Resource**. What does the principal want to perform the action on?
+
+Each of these three elements can have limited values in the context of the Airflow environment.
+You can see the list of principals, actions and resources in the policy store schema by following the steps below.
+
+1. Open the `Amazon Verified Permissions console <https://console.aws.amazon.com/verifiedpermissions>`_.
+2. Choose the policy store used by Airflow (by default its description is ``Airflow``).
+3. In the navigation pane on the left, choose **Schema**.
+
+Example of policies
+-------------------
+
+Here are some example of policies you can define in Amazon Verified Permissions.
+You can use them as-is if they fit exactly your use case.
+You can also modify and/or combine them to create your owned tailor made policies.
+
+Give all permissions to specific user
```

Comment:
Thinking more about it, it would best if we give example of what would existing FAB roles (Admin, Op, User, Viewer, Public) be written as when the Airflow deployment manager using AWS Auth manager. Current examples cover Admin and Viewer only. Add these will allow deployment managers to easily achieve what's possible today and then modify from there.

Reply:
Good point. I'll create a PR to cover these.

Reply:
#39040